src/dist/app_daemon/daemon.server.cpp

```diff
@@ -522,7 +522,11 @@ namespace dsn
             if (!app->exited && app->process_handle)
             {
 # ifdef _WIN32
-                ::TerminateProcess(app->process_handle, 0);
+                std::ostringstream pid;
+                std::string command;
+                pid << GetProcessId(app->process_handle);
+                command = "TASKKILL /F /T /PID " + pid.str();
+                system(command.c_str());
```

Comment:
This is great, and I'll merge the code immediately. However, I'm wondering is there any windows API we can use instead of invoking an external command?